### ISSUE TITLE
query("@fadeInPicker")` returned zero elements. 

### ISSUE THREAD

**Issue by klonq123** [DanielYKPan/date-time-picker#406]
The programs compies and runs perfectly in dev mode.

When running in production mode with Angular CLI

ng build --prod

The following stack trace occurs when I try to show the popup by selecting the input.

**Dependecies**

  "dependencies": {
    "@angular/router": "^6.0.7",
    "@angular/platform-browser-dynamic": "^6.0.7",
    "@angular/platform-browser": "^6.0.7",
    "@angular/http": "^6.0.7",
    "@angular/forms": "^6.0.7",
    "@angular/compiler": "^6.0.7",
    "@angular/cdk": "^6.3.2",
    "@angular/animations": "^6.0.7",
    "@angular/common": "^6.0.7",
    "@angular/core": "^6.0.7",
    "angular2-flash-messages": "^2.0.5",
    "core-js": "^2.5.4",
    "highcharts": "^6.1.0",
    "ng-pick-datetime": "^6.0.8",
    "rxjs": "^6.2.0",
    "rxjs-compat": "^6.2.0",
    "zone.js": "^0.8.26"
  },
  "devDependencies": {
    "@types/node": "^10.5.1",
    "@types/highcharts": "^5.0.23",
    "@angular/language-service": "^6.0.7",
    "@angular/compiler-cli": "^6.0.7",
    "@angular-devkit/build-angular": "~0.6.8",
    "@angular/cli": "~6.0.8",
    "@types/jasmine": "^2.8.8",
    "@types/jasminewd2": "~2.0.3",
    "ajv": "^6.5.0",
    "codelyzer": "~4.2.1",
    "jasmine-core": "~2.99.1",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "^2.0.2",
    "karma-chrome-launcher": "~2.2.0",
    "karma-coverage-istanbul-reporter": "~2.0.0",
    "karma-jasmine": "~1.1.1",
    "karma-jasmine-html-reporter": "^0.2.2",
    "protractor": "~5.3.0",
    "ts-node": "~5.0.1",
    "tslint": "~5.9.1",
    "typescript": "~2.7.2"
  }

**Stack trace :**
Uncaught Error: Unable to process animations due to the following failed trigger transitions
 @transformPicker has failed due to:

- `query("@fadeInPicker")` returned zero elements. (Use `query("@fadeInPicker", { optional: true })` if you wish to allow this.)

    at t.reportError (main.cf3efbaad8072f78399e.js:1)
    at t._flushAnimations (main.cf3efbaad8072f78399e.js:1)
    at t.flush (main.cf3efbaad8072f78399e.js:1)
    at e.t.flush (main.cf3efbaad8072f78399e.js:1)
    at main.cf3efbaad8072f78399e.js:1
    at e.invoke (polyfills.abb0c25ad00b471fb928.js:1)
    at t.run (polyfills.abb0c25ad00b471fb928.js:1)
    at t.runOutsideAngular (main.cf3efbaad8072f78399e.js:1)
    at t.end (main.cf3efbaad8072f78399e.js:1)
    at t.detectChanges (main.cf3efbaad8072f78399e.js:1)

**Comment by DanielYKPan** [comment]
The reason you have such an error may because you use the picker inappropriately . You should test your component before building your app.

**Comment by klonq123** [comment]
@DanielYKPan 
This error is gone, I updated to 6.0.10 and I uncommented to following line in the default polyfills.ts file created by Angular CLI :
import 'web-animations-js';  // Run `npm install --save web-animations-js`.

Even though I am using chrome (latest). It might be worth noting the polyfills in the documentation.

All is not well though, now I am running into [this issue](https://github.com/angular/angular-cli/issues/2588). Again, the problem is only present when using the **--prod** operator.
